packages/online-editor/static/index.html

```diff
@@ -21,20 +21,27 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta
+      name="description"
+      content="DMN.new and BPMN.new let you draw Decision Model and Notation(DMN), Predictive Model Markup Language(PMML) and Business Process Model and Notation(BPMN) diagrams for your business processes"
```

Comment:
Just small suggestion. I would add space before the left parenthesis and add the dot at the end of the sentence.
```suggestion
      content="DMN.new and BPMN.new let you draw Decision Model and Notation (DMN), Predictive Model Markup Language (PMML) and Business Process Model and Notation (BPMN) diagrams for your business processes."
```